docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -78,45 +78,85 @@ These instructions assume you have experience using the command line, understand
 >
 > If debugging stops working; for example, if breakpoints are being ignored; stop debugging. Then, if necessary, close all host application windows and the Node window. Finally, close Visual Studio Code and reopen it.
 
-### Appendix A
+## Debug a project not created with Yo Office
 
 If your project wasn't created with Yo Office, you need to create a debug configuration for Visual Studio Code.
 
+### Configure package.json file
+
+1. Ensure you have a `package.json` file. If you don't already have a package.json file, run `npm init` in the root folder of your project and answer the prompts.
+1. Run `npm install office-addin-debugging`. This package will sideload your add-in for debugging.
+1. Open the package.json file. In the `scripts` section add the following script.
+
+   ```json
+   "start:desktop": "office-addin-debugging start $MANIFEST_FILE$ desktop",
+   "dev-server": "$SERVER_START$"
+   ```
+
+1. Replace `$MANIFEST_FILE$` with the correct file name and folder location of your manifest.xml file.
```

Comment:
Since the previous parts of the instruction allude to this file name being flexible, consider not using the standard file name here.
```suggestion
1. Replace `$MANIFEST_FILE$` with the correct file name and folder location of your manifest.
```